### PR TITLE
added information for update properties highlighted.

### PR DESCRIPTION
**Pull request by irenesmith**
Modified support for the update values highlighted as of Firefox 63.

### CODE REVIEW

File: webextensions/api/tabs.json [hunk 1]
```diff
@@ -2917,7 +2917,7 @@
                     "version_added": false
                   },
                   "firefox": {
-                    "version_added": false
+                    "version_added": 63
```
Comment (Elchi3) [inline webextensions/api/tabs.json]: Needs to be a string `"63"`.

**Comment by Elchi3** [comment]
This was added in https://hg.mozilla.org/mozilla-central/rev/4e78d806cb63, https://bugzilla.mozilla.org/show_bug.cgi?id=1464862

From the diff it also seems this touched android code, so maybe set firefox_android to 63 as well? 
Also, https://bugzilla.mozilla.org/show_bug.cgi?id=1464862#c9 says "be sure this platform difference is documented on mdn correctly." I'm not sure if this means it is not supported on android or if the implementation is different there.

**Comment by irenesmith** [comment]
Checking into what support for Android should be.

**Comment by irenesmith** [comment]
This is correct. Firefox for Android does not support this property because there is no support for multi-tab selections.

**Comment by Loirooriol** [comment]
Just want to point out that `updateProperties.highlighted` was added in https://bugzilla.mozilla.org/show_bug.cgi?id=1479129, not https://bugzilla.mozilla.org/show_bug.cgi?id=1464862. So as Irene said this wasn't implemented for Android, the PR was right.

**Pull request merged**